modules/ROOT/pages/visual-debugger-concept.adoc

```diff
@@ -17,18 +17,38 @@ When you run your application in debug mode for the first time, Anypoint Studio
 
 To debug your application in Studio:
 
-. Right-click your project in the *Package Explorer* view.
-. Select *Debug As* > *Mule Application*. +
-Studio asks you for permission to switch to the *Mule debug* perspective.
+. Right-click your project in *Package Explorer*.
+. Select *Debug As* > *Mule Application*.
++
+Studio requests permission to switch to the *Mule debug* perspective.
 . Click *Yes*.
 
-Additionally, you can run your application in debug mode by right-clicking in the canvas and selecting *Debug Project*, or by clicking the *Debug* button on your top toolbar image:to-run-debug-mode-e7b7b.png[debug-button].
+Additionally, run your application in debug mode by right-clicking in the canvas and selecting *Debug Project*, or by clicking the *Debug* button on the toolbar image:to-run-debug-mode-e7b7b.png[debug-button].
 
 == Configure Studio Debugger
 
-When debugging in Studio, the debugger listens for incoming TCP connections on `localhost` and port `6666`. You can modify this configuration in by clicking *Run* > *Debug configurations* and selecting the *Debug* tab.
+Enable precise control over application debugging by setting the debugger connections and maximum values. To modify the default configuration values, click *Run* > *Debug configurations* and select the *Debug* tab.
+
+[%header,%autowidth.spread,cols="a,a,a"]
+|===
+| Option | Description | Default Value
+
+| Connection Properties 
+| Studio debugger listens to incoming TCP connections on `localhost` and port `6666`. Pause the debugger listening activities by selecting *Suspend* to prevent the debugger from triggering or processing any incoming requests temporarily. 
+| 
+
+* TCP connections: `localhost`
+* Port: `6666`
```

Comment:
These bullets duplicate the first sentence in the description. Any way to address?

Reply:
Let's see if my wording works.